src/bin/uutils.rs

```diff
@@ -9,74 +7,84 @@
  * file that was distributed with this source code.
  */
 
+// spell-checker:ignore (acronyms/names) Gehring
+// spell-checker:ignore (rustlang/crates) clippy concat rustlang termwidth textwrap
+// spell-checker:ignore (uutils) coreutils sigpipe uucore uumain uutils
+// spell-checker:ignore (shell) busybox symlinks
+
 include!(concat!(env!("OUT_DIR"), "/uutils_crates.rs"));
 
+extern crate lazy_static;
+extern crate textwrap;
+extern crate uucore;
+
+use lazy_static::lazy_static;
 use std::collections::hash_map::HashMap;
 use std::io::Write;
-use std::path::Path;
 
-extern crate uucore;
-
-static NAME: &str = "uutils";
 static VERSION: &str = env!("CARGO_PKG_VERSION");
 
+lazy_static! {
+    static ref BINARY_PATH: std::path::PathBuf = {
+        // support symlinks by using args[0], when possible, with fallback to current_exe()
+        match std::env::args().next() {
+            Some(ref s) if !s.is_empty() => std::path::PathBuf::from(s),
+            _ => std::env::current_exe().unwrap(),
+        }
+    };
+    static ref NAME: &'static str = &*BINARY_PATH.file_stem().unwrap().to_str().unwrap();
+}
+
 include!(concat!(env!("OUT_DIR"), "/uutils_map.rs"));
 
-fn usage(cmap: &UtilityMap) {
-    println!("{} {}", NAME, VERSION);
-    println!();
-    println!("Usage:");
-    println!("  {} [util [arguments...]]\n", NAME);
-    println!("Currently defined functions:");
+fn usage(utils: &UtilityMap) {
+    println!("{} {} (multi-call binary)\n", *NAME, VERSION);
+    println!("Usage: {} [function [arguments...]]\n", *NAME);
```

Comment:
I would personally prefer `utility`, but I'm okay with this.

Reply:
Not personally attached to this usage... I'm using "function/utility" in other error/help text. I think one or the other but not both to minimize confusion. And I would tend toward this as it is the `busybox` terminology. But if you have a stronger like for "utility", let me know and I'll change it.